operator/src/test/java/org/keycloak/operator/ClusterOperatorTest.java

```diff
@@ -158,6 +160,9 @@ protected static void deployDB() {
     Log.info("Checking Postgres is running");
     Awaitility.await()
             .untilAsserted(() -> assertThat(k8sclient.apps().statefulSets().inNamespace(namespace).withName("postgresql-db").get().getStatus().getReadyReplicas()).isEqualTo(1));
+
+    // Deploy creds secret
+    k8sclient.secrets().inNamespace(namespace).createOrReplace((Secret) getResourceFromMultiResourceFile("example-keycloak.yml", 1));
```

Comment:
nitpick, I think we can delegate this "extraction" to it's own function, not fussy though.

Reply:
👍